test/reporters/xunit.spec.js

```diff
@@ -350,6 +350,46 @@ describe('XUnit reporter', function() {
           '</failure></testcase>';
         expect(expectedWrite, 'to be', expectedTag);
       });
+
+      it('should handle non-string diff values', function() {
```

Comment:
You have to emit an EVENT_TEST_FAIL to run this test successfully.
var runner = new EventEmitter();